sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Resources.Designer.cs

```diff
@@ -65,5 +65,17 @@ internal Resources()
                 resourceCulture = value;
             }
         }
+
+        /// <summary>
+        ///   The specified container or blob does not exist
+        /// </summary>
+        internal static string ResourceNotFound
+        {
+            get
+            {
+                return ResourceManager.GetString("ResourceNotFound", resourceCulture);
+            }
+        }
```

Comment:
I think you added this manually (correct me if I'm wrong). You should add it to the `Resources.resx` file, and `Resources.Designer.cs` is generated automatically. If someone triggers the file generation, it will overwrite your changes here.

You can take a look in the `Resources.resx` file in the main `Azure.Messaging.EventHubs` package as a reference.

Reply:
yes, you're right :) I didn't know about that.